frontend_next/components/caseStudy/RelevantCaseStudies.tsx

```diff
@@ -0,0 +1,48 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import { caseStudyCard } from "@/constants";
+
+const RelevantCaseStudies = () => {
+  return (
+    <section className="bg-1">
+      <h2 className="caseSubtitle">Projects</h2>
+      <h3 className="caseTitle">Other Case Studies</h3>
+
+      {/* card */}
+      <div className="mt-9 flex flex-wrap gap-6 md:mt-10">
+        {caseStudyCard.map((item) => (
+          <div
+            key={item.title}
+            className="dark:bg-black-300 rounded-[20px] bg-white shadow-[0px_1px_20px_0px_#E0ECFF] dark:shadow-[0px_3px_30px_0px_#1F2C41]  md:max-w-[430px]"
+          >
+            <Image
+              src={item.image}
+              alt="project image"
+              width={0}
+              height={0}
```

Comment:
It's important for the browser when rendering the image to know the size of the images. Have you ever been on a website and as its loading all the elements are shifting around and moving? This can be a cause of that. Set the correct dimensions so the browser knows what space to make for the element as it loads in.